miso-web/src/main/webapp/scripts/list_servicerecord.js

```diff
@@ -66,18 +78,28 @@ ListTarget.servicerecord = {
         bSortable: true,
         mRender: function (data, type, full) {
           if (type === "display") {
-            return data
-              ? '<a href="' +
-                  Urls.ui.instruments.editRecord(config.instrumentId, full.id) +
-                  '">' +
-                  data +
-                  "</a>"
-              : "";
+            if (config.instrumentId) {
+              return data
+                ? '<a href="' +
+                    Urls.ui.instruments.editRecord(config.instrumentId, full.id) +
+                    '">' +
+                    data +
+                    "</a>"
+                : "";
+            } else {
+              return data
+                ? '<a href="' +
+                    Urls.ui.freezers.editRecord(config.freezerId, full.id) +
+                    '">' +
+                    data +
+                    "</a>"
+                : "";
+            }
```

Comment:
Reduce duplication by setting url in a variable: `var url = config.instrumentId ? Urls.ui.instruments... : Urls.ui.freezers...`